PubnativeLite/PubnativeLite/HyBidSettings.m

```diff
@@ -104,6 +105,17 @@ - (NSString *)orientation {
     }
 }
 
+- (NSString *)deviceSound {
+    [[AVAudioSession sharedInstance] setActive:YES error:nil];
+    if ([AVAudioSession sharedInstance].outputVolume == 0) {
+        [[AVAudioSession sharedInstance] setActive:NO error:nil];
+        return @"0";
+    } else {
+        [[AVAudioSession sharedInstance] setActive:NO error:nil];
```

Comment:
Why sou set AVAudioSession active to NO in the else case? 
And isn't necessary to set it all? I think checking the output volume is enough

Reply:
@faresbh Without setting to `YES`, the `outputVolume` always returns the same (initial) value. After reading the `outputVolume` value, I deactivate the session because I thought that'd be the smart thing to do since we've set the session to become active and we wont be needing it until next request.